.gitignore

```diff
@@ -43,3 +43,6 @@ jspm_packages
 
 # Webstorm project metadata
 .idea
+
+# macOS
```

Comment:
See comment on other PR:

https://coderwall.com/p/vz6ymw/how-to-globally-gitignore-ds_store